ToDoApp/containers/home/Home.js

```diff
@@ -1 +1,90 @@
-//File
+import React, { Component } from 'react';
+import {TouchableOpacity, Text, View, ScrollView, StyleSheet } from 'react-native';
+import {createStackNavigator, createAppContainer} from 'react-navigation';
+import Item from '../../components/item/Item';
+import ButtonCustom from '../../components/button/ButtonCustom';
+import Detail from '../detail/Detail';
+import NewTask from '../newTask/NewTask';
+
+var data = [
```

Comment:
maybe we could extract this mocked data to another file

Reply:
This is dummy data, now I have deleted that because in real application it will not be used. Do you agree?

Reply:
Sorry, that I commented in the last message will be included in a following pull request in which I change the way how I handle the data of the component.